application/app/views/projects/show/_project_tabs.html.erb

```diff
@@ -1,5 +1,5 @@
 <!-- Project Tabs -->
-<div id="project-tabs" class="mb-3">
+<div id="projects project-tabs" class="mb-3">
```

Comment:
I think we shouldn't use a div id with two words separated by whitespace. Could you join the string?

Reply:
fixed